tiatoolbox/cli.py

```diff
@@ -66,7 +66,10 @@ def main():
     "the meta information, default=show",
```

Comment:
Did you add a cli function for stain normalisation? This needs to be added and also need to write a test for it.

Reply:
No, I didn't add that. I Will do it.